### PR TITLE
📝 Small improvements for beginners

### PR DESCRIPTION
**Pull request by CollierCZ**
## Why

A new user gave some feedback about some things that were unclear. See Context.

## What's changed

Tried to add a little clarity.

### CODE REVIEW

File: docs/src/get-started/deploy/init.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ Then go through each of the steps to create the project:
    The CLI lists each region's location, cloud provider, and average carbon intensity.
 3. Choose a plan.
    A Development plan is enough before you deploy anything.
+   You can't choose production plans during a free trial.
```
Comment (lolautruche) [inline docs/src/get-started/deploy/init.md:27]: Should it be mentioned that, as a consequence, a free trial user won't have to choose a plan?

**Review by lolautruche — CHANGES_REQUESTED** [review]

File: docs/src/create-apps/app-reference.md [hunk 2]
```diff
@@ -51,6 +51,9 @@ use the [`source.root` property](#source).
 ## Types
 
 The `type` defines the base container image used to run the application.
+The version is the major and sometimes minor version numbers.
```
[the remark this answers is not in the document]
Comment (CollierCZ) [reply docs/src/create-apps/app-reference.md:54]: I wanted to, it's just some versions have `X.Y` and some have only `X`. There are the examples in a table right below this: https://various-du22ssa-652soceglkw4u.eu-3.platformsh.site/create-apps/app-reference.html#types

File: docs/src/get-started/deploy/init.md (hunk 1 above)
Comment (CollierCZ) [reply docs/src/get-started/deploy/init.md:27]: That was the intent. I'll try to improve it.

Comment (CollierCZ) [reply docs/src/get-started/deploy/init.md:27]:
```suggestion
   During a free trial, you can't choose production plans so you won't see this option.
```

File: docs/src/create-apps/app-reference.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (lolautruche) [reply docs/src/create-apps/app-reference.md:54]: I understand, but many people may verlook at the examples table. 
Here is what I'd suggest:
```suggestion
The version is the **major** and sometimes **minor** version numbers, e.g. `X` or `X.Y` (see the examples table below).
```

**Review by lolautruche — CHANGES_REQUESTED** [review]

File: docs/src/create-apps/app-reference.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (CollierCZ) [reply docs/src/create-apps/app-reference.md:54]:
```suggestion
The version is the major (`X`) and sometimes minor (`X.Y`) version numbers,
depending on the service, as in the following table.
```

**Review by lolautruche — APPROVED** [review]

**Pull request merged**